### PR TITLE
[TASK] Polish the class and file comments

### PR DESCRIPTION
**Pull request by oliverklee**
- add a class comment for the demo class
- improve the test case comments
- remove a misleading "part of the TYPO3 project" file comment

### CODE REVIEW

File: Tests/Unit/FirstClassTest.php [hunk 1]
```diff
@@ -4,6 +4,9 @@
 use Helhum\ExtScaffold\Tests\Unit\Fixtures\LoadableClass;
 use Nimut\TestingFramework\TestCase\UnitTestCase;
 
+/**
+ * Test case.
```
Comment (helhum) [inline Tests/Unit/FirstClassTest.php]: that comment does not provide any value, I'd rather skip it

File: Tests/Functional/RenderingTest.php [hunk 2]
```diff
@@ -1,39 +1,13 @@
...
- ***************************************************************/
-
 use Nimut\TestingFramework\Http\Response;
 use Nimut\TestingFramework\TestCase\FunctionalTestCase;
 use PHPUnit\Util\PHP\AbstractPhpProcess;
 use TYPO3\CMS\Core\Utility\GeneralUtility;
 
 /**
- * Class RenderingTest
+ * Test case.
```
Comment (helhum) [inline Tests/Functional/RenderingTest.php]: same here. rather remove the complete comment

File: Tests/Unit/FirstClassTest.php (hunk 1 above)
Comment (oliverklee) [reply Tests/Unit/FirstClassTest.php]: Done.

File: Tests/Functional/RenderingTest.php (hunk 2 above)
Comment (oliverklee) [reply Tests/Functional/RenderingTest.php]: Done.

**Comment by oliverklee** [comment]
Updated and repushed.

**Comment by helhum** [comment]
thanks

**Pull request merged**